examples/run/gcs-resource-spec-taskrun.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: pipeline.knative.dev/v1alpha1
+kind: TaskRun
+metadata:
+  name: list-file
+spec:
+  taskSpec:
+   inputs:
+    resources:
+    - name: workspace
+      type: storage
+      targetPath: rules
```

Comment:
btw [`targetPath`](https://github.com/knative/build-pipeline/blob/master/docs/tasks.md#controlling-where-resources-are-mounted) isn't required, this is optional for cases where folks want to explicitly override [the defaults](https://github.com/knative/build-pipeline/blob/master/docs/tasks.md#input-resources)